esrally/driver/runner.py

```diff
@@ -65,6 +65,11 @@ def register_default_runners():
     register_runner(track.OperationType.CreateSnapshotRepository.name, Retry(CreateSnapshotRepository()), async_runner=True)
     register_runner(track.OperationType.WaitForRecovery.name, Retry(IndicesRecovery()), async_runner=True)
     register_runner(track.OperationType.PutSettings.name, Retry(PutSettings()), async_runner=True)
+    register_runner(track.OperationType.CreateTransform.name, Retry(CreateTransform()), async_runner=True)
```

Comment:
Could you please add documentation for thenew operations in [track.rst](https://github.com/elastic/rally/blob/master/docs/track.rst#operations)?